agenthub/codeact_swe_agent/README.md

```diff
@@ -0,0 +1,7 @@
+# CodeAct (SWE Edit Specialized)
+
+This agent is an adaptation of the original [SWE Agent](https://swe-agent.com/) based on CodeAct using the `agentskills` library of OpenDevin.
+
+It is intended use is **solving Github issues**.
+
+It removes web-browsing and Github capability from the original CodeAct agent to avoid confusion to the agent.
```

Comment:
Bc we want this agent to solve focus on solving github issues - intentionally keep the scope small so hopefully it give us more performance.

Reply:
Removing GitHub capability definitely makes sense. At least it could save some token cost XD.

Removing web-browsing sort of makes sense too - if we know it's never used in the benchmark (hey, we probably need better OBSERVABILITY for OpenDevin! Right now we only have metrics for cost). In fact, it might be a good idea to remove it for fair comparison - otherwise CodeActAgent might as well search for the issue, find the right PR, and cheats by copying from the PR.